staging/src/k8s.io/kubelet/config/v1beta1/types.go

```diff
@@ -64,9 +64,12 @@ const (
 	// NoneTopologyManager Policy is a mode in which kubelet has no knowledge
 	// of NUMA alignment of a pod's CPU and device resources.
 	NoneTopologyManagerPolicy = "none"
-	// SingleNumaNodeTopologyManager Policy iis a mode in which kubelet only allows
+	// SingleNumaNodeTopologyManager Policy is a mode in which kubelet only allows
 	// pods with a single NUMA alignment of CPU and device resources.
 	SingleNumaNodeTopologyManager = "single-numa-node"
+	// NoneMemoryManagerPolicy is a memory manager none policy, under the none policy
+	// the memory manager will not pin containers memory of guaranteed pods
+	NoneMemoryManagerPolicy = "none"
```

Comment:
where is the static or single-numa-node policy token?

Reply:
This constant only used to set the default value for the `MemoryManagerPolicy`(see https://github.com/kubernetes/kubernetes/pull/95479/commits/951d7975dc99d8ff59d43f5821c4b1b1c1183b6e#diff-64312739de7a7da86a275e2ada9e0caf61e826c20b90ffbb6c1bafa464d15fcfR158), I did not add a static one, because currently it no use for it.

Reply:
Ok I will add it

Reply:
See https://github.com/kubernetes/community/blob/master/contributors/devel/sig-architecture/api-conventions.md#constants - we've clarified that this approach for constants is wrong and the new approach is documented.  See my comment above about how to reconcile it, but public API constants must be `CamelCase`.